components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/JWTTokenIssuer.java

```diff
@@ -102,6 +102,8 @@ public class JWTTokenIssuer extends OauthTokenIssuerImpl {
 
     private static final String AUTHORIZATION_PARTY = "azp";
     private static final String CLIENT_ID = "client_id";
+    private static final String APP_DOMAIN = "app_domain";
+    private static final String USER_DOMAIN = "user_domain";
```

Comment:
shall we have the word tenant domain in the claim. specially when using user_domain this could be confused with userstore domian.
some suggetions, 
app_domain -> app_td
user_domain -> user_td

Reply:
changed as suggested